extra/enterprise.rst

```diff
@@ -0,0 +1,87 @@
+:orphan:
+
+Installing Citus Enterprise Edition
+===================================
+
+.. _enterprise_debian:
+
+Ubuntu or Debian
+----------------
+
+1. Contact us to obtain a Citus Enterprise repository token. This token grants download access to the packages.
+
+2. Create a package tarball:
```

Comment:
Hey Joe, this is missing a few items we discussed:
- Two methods: If you have internet, or if you don't. This only captures if you don't have internet.
- Upgrading from citus community. We should document steps needed for that.